provider/docker.go

```diff
@@ -614,11 +621,28 @@ func listServices(ctx context.Context, dockerClient client.APIClient) ([]dockerD
 	}
 
 	var dockerDataList []dockerData
+	var dockerDataListTasks []dockerData
 
 	for _, service := range serviceList {
 		dockerData := parseService(service, networkMap)
 
-		dockerDataList = append(dockerDataList, dockerData)
+		sticky, _ := strconv.ParseBool(provider.getSticky(dockerData))
```

Comment:
I did this because you need to automatically bypass the swarm LB if you want to use sticky sessions.  Basically if you explicitly set the flag to bypass or you are using sticky sessions then you should bypass the swarm LB.